styles.css

```diff
@@ -11,24 +11,30 @@ diplay: inline-block;
     box-sizing: border-box;
 }
 
-body {
-  /*   font: ; 
-        color:
-
-  */
+body{
+padding: 50px 0 0;
+font: 14px;
+color: #959595;
+background-color: #ffffff;
+}
+header{
+    color: #959595;
+    background: #222222;
+}
+.menu-container{
+    display: flex;
+    justify-content: flex-start;
+    margin: 0 0 20px;
+    border: 1px;
 }
-element.style{
-    background: black;
-}
-
 ul {
```

Comment:
element selectors should contain styles that are common for whole page, try to use class